Notebooks/.OwlBot.yaml

```diff
@@ -1,4 +1,4 @@
 deep-copy-regex:
-    - source: /google/cloud/notebooks/(v1|v1beta1|v2)/.*-php/(.*)
+    - source: /google/cloud/notebooks/(v2)/.*-php/(.*)
```

Comment:
Hope this change is okay?

Reply:
cc: @bshaffer

Reply:
Hello! This is not okay, as this will stop updating the `v1` and `v1beta1` components of the Notebook API, which we currently have deployed. Please revert this back!

Reply:
```suggestion
    - source: /google/cloud/notebooks/(v1|v1beta1|v2)/.*-php/(.*)
```